.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+## Related issue
+- One line for each issue. If merging this pull request should auto-close the issue on that line,
```

Comment:
You've added list formatting to all of these sections, which is unnecessary. Just make them all regular paragraphs by removing the preceding `-` character.

Only the "Checklist" section needs to be a list.